### ISSUE TITLE
Build fails with Go 1.5

### ISSUE THREAD

**Issue by marten-seemann** [lucas-clemente/aes12#1]
On a `Linux scw-c1939b 4.5.1-docker-1 #1 SMP Tue Apr 19 12:55:15 UTC 2016 x86_64 x86_64 x86_64 GNU/Linux` building fails:

```
# github.com/lucas-clemente/aes12
../aes12/gcm_amd64.s:348: unrecognized instruction "PINSRB"
../aes12/gcm_amd64.s:407: unrecognized instruction "PINSRB"
../aes12/gcm_amd64.s:895: unrecognized instruction "PINSRB"
../aes12/gcm_amd64.s:1267: unrecognized instruction "PEXTRB"
asm: asm: assembly of ../aes12/gcm_amd64.s failed
```

**Comment by lucas-clemente** [comment]
Which go version is this?

**Comment by marten-seemann** [comment]
go version go1.5.1 linux/amd64

**Comment by lucas-clemente** [comment]
Then I would say this is expected behavior, we don't currently support 1.5 (or even test it on the CI). Do you think we should?

**Comment by marten-seemann** [comment]
Well, this is a different package from our quic-go. But let me try to run it on a newer Go version on my vServer first.

**Comment by lucas-clemente** [comment]
The package is question is the AES-GCM implementation I rigged for quic-go ;) It uses vector instructions, which are only available from 1.6 and up.
